packages/bbui/src/Button/Button.svelte

```diff
@@ -32,7 +32,7 @@
     class:spectrum-Button--cta={cta}
     class:spectrum-Button--primary={primary}
     class:spectrum-Button--secondary={secondary}
-    class:spectrum-Button--warning={warning}
+    class:spectrum-Button--warning={warning && !disabled}
```

Comment:
Why is this not the case with the case in the other styles, like `secondary`?

Reply:
Oh maybe I need to add a custom disabled style. Let me look